SCF/SCF-session1/Assignment-1/ShopingCart.java

```diff
@@ -0,0 +1,213 @@
+import java.util.*;
+/**represent the Shoping Cart
+ * @author Ramdev
+ * this class have an another class "Item",
+ */
+public class ShopingCart 
+{
+	public static void main(String[] args) 
+	{
+		Scanner input = new Scanner(System.in);
+		ShopingCart cart = new ShopingCart();
+		int choice = 0,ID = 0,quantity;
+		boolean flag = true;
```

Comment:
I guess we can think of a better variable name than "flag". Something like "displayMenu" would have been a better choice